libs/expo/betterangels/src/lib/screens/Interactions/InteractionsSorting.tsx

```diff
@@ -87,10 +87,10 @@ export default function InteractionsSorting(props: IInteractionsSortingProps) {
 
 const styles = StyleSheet.create({
   sortButton: {
-    height: 40,
-    width: 40,
+    height: Spacings.xl,
+    width: Spacings.xl,
```

Comment:
I think I actually talked myself out of it in another ticket. I don't think these numbers _are_ magic numbers, strictly speaking, because their meaning is obvious. So, we can leave them as is